saleor/app/management/commands/create_app.py

```diff
@@ -59,9 +59,9 @@ def handle(self, *args: Any, **options: Any) -> Optional[str]:
         permissions = clean_permissions(permissions)
         app = App.objects.create(name=name, is_active=is_active)
         app.permissions.set(permissions)
-        token_obj = app.tokens.create()
+        _token_obj, auth_token = app.tokens.create()
```

Comment:
```suggestion
        _,  auth_token = app.tokens.create()
```
We don't use `_token_obj`